pandas/core/groupby/groupby.py

```diff
@@ -2312,26 +2312,42 @@ def _get_cythonized_result(
             return self._wrap_transformed_output(output)
 
     @Substitution(name="groupby")
-    @Appender(_common_see_also)
     def shift(self, periods=1, freq=None, axis=0, fill_value=None):
         """
         Shift each group by periods observations.
 
+        When freq is not passed, shift the index without realigning the data.
```

Comment:
I think should remove this first line (second is OK to keep)

Reply:
okay